### ISSUE TITLE
Wave view of newly opened file empty until window resize

### ISSUE THREAD

**Issue by humdingerb** [HaikuArchives/BeAE#33]
After you opened a sound file or d&d'ed one, the wave view appears to be empty. Only after resizing the window, it is drawn.

**Comment by janus2** [comment]
@humdingerb  did you have the old version installed?

**Comment by humdingerb** [comment]
Nope. newly compiled the master. (Currently trying to fix some minor drawing issue in the prefs list.)

**Comment by janus2** [comment]
This is strange I don't see this problem at all...
gcc2?

**Comment by janus2** [comment]
The localization of the app is even more complex even the shortcut are in a lookup table related to the language

**Comment by janus2** [comment]
You don't have an old version installed because when you open a sample the old one can be used.

**Comment by humdingerb** [comment]
Only have the executable I built myself (current master). 32bit hrev52355.
